geospatial/src/Dem_TEST.cc

```diff
@@ -282,3 +282,22 @@ TEST_F(DemTest, LunarDemLoad)
   EXPECT_NEAR(dem.WorldWidth(), 80.0417, 1e-2);
   EXPECT_NEAR(dem.WorldHeight(), 80.0417, 1e-2);
 }
+
+TEST_F(DemTest, LargeVRTWithZipAndCurl)
+{
+  // Load a large VRT DEM from another server (used by ArduPilot).
+  common::Dem dem;
+  dem.SetXSizeLimit(100);
+  dem.SetYSizeLimit(50);
+  //! @todo Make a really large VRT locally
+  auto const res = dem.Load("/vsizip/vsicurl/https://terrain.ardupilot.org/SRTM1/ap_srtm1.zip/ap_srtm1.vrt");
```

Comment:
I don't think we should be accessing ardupilot.org from a test here. I'd rather avoid accessing the internet if possible, otherwise, maybe we can add something to Fuel?

Reply:
Can do

Reply:
Can you upload the following file into the Fuel for me? You can call it `S36E149.hgt.zip`. 

https://terrain.ardupilot.org/SRTM1/S36E149.hgt.zip
    
```bash
$ gdallocationinfo -wgs84 /vsizip/vsicurl/https://terrain.ardupilot.org/SRTM1/ap_srtm1.zip/ap_srtm1.vrt 149.165230 -35.363261
Report:
  Location: (1184995P,429708L)
  Band 1:
    <LocationInfo><File>/vsizip/vsicurl/https://terrain.ardupilot.org/SRTM1/S36E149.hgt.zip/S36E149.hgt</File></LocationInfo>
    Value: 587
```

Reply:
Done in c001ba176f7e199ee6276581e8c668ef8b5b4b2b. It works without internet. I added both a VRT and a VRT inside of a zipfile to test both the /vsizip and large VRT scenarios as part of the repo. It doesn't use Fuel.